### DISCUSSION TITLE
Occasional Left Mouse Unresponsive with Catalina. Resolution?

### DISCUSSION THREAD

**Discussion by jaredev** [yusifsalam/t490-macos#48] [Q&A]
Hello. Thanks again for the very useful configuration!

I've been using the Catalina version and am considering upgrading to BigSur soon. The only issue that I've had is that the track point occasionally becomes unresponsive, sometimes the left click on both the buttons, trackpad, and mouse will become unresponsive and require a restart. It happens intermittently and I haven't found a specific cause yet.

Has anyone else had this issue? Is there a resolution? And if this is a known issue, is it fixed in the newest release for BigSur?

Thanks again for your help!

**Comment by yusifsalam** [comment]
Have you updated VoodooRMI yourself? The one included in the Catalina release is quite outdated although I don't recall ever having that specific issue myself.

**Reply by jaredev** [reply]
Thanks for your rapid response! I got it booting again after updating to VoodooPS2Controller 2.2.5, and VoodooRMI 2.2.5 release kexts. So far so good, but the issue was intermittent so I should know for sure if that resolves it within a few days, as it was occurring at least once a day prior.

It did require some minor changes to the boot config. If all goes well, I can upload the new configuration and files for Catalina if you want.

Thanks again!

**Comment by jaredev** [comment]
For an update. I've been using the new kexts for about a week. The issue is happening less often, and when it does occur, seems to only last a few minutes. I confirmed it wasn't related to system resource usage, as I could still use the keyboard and navigate in the OS. It seems to have improved, where it resolves itself somehow without requiring a reboot as before.

I may do a Big Sur clean install and see if that resolves it completely.

Thanks again for your help.
